ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
So, is this something related to obtaining a dummy object where you can set attributes? If yes, this is something that has to be changed as it confuses the new developers coming in. We can address this separately.